code/_onclick/hud/screen_objects.dm

```diff
@@ -169,7 +169,7 @@
 /obj/screen/storage/Click(location, control, params)
 	if(world.time <= usr.next_move)
 		return 1
-	if(usr.stat || usr.paralysis || usr.stunned || usr.IsWeakened())
+	if(usr.incapacitated())
```

Comment:
Change in functionality; this needs to ignore lying down or restraints.

Reply:
Whoops, fixed